src/release_notes_workflow/release_notes_check_args.py

```diff
@@ -40,11 +41,50 @@ def __init__(self) -> None:
             "--output",
             help="Output file."
         )
+        
+        # AI-powered release notes options
+        parser.add_argument(
+            "--ai",
+            action="store_true",
+            help="Generate AI-powered release notes using AWS Bedrock"
+        )
+        parser.add_argument(
+            "--ai-model",
+            default="claude-3-7-sonnet",
+            help="AI model to use for analysis (default: claude-3-7-sonnet)"
+        )
+        parser.add_argument(
+            "--baseline-date",
+            help="Baseline date for commit analysis (format: YYYY-MM-DD)"
+        )
```

Comment:
The `--date` on line 35 can be the baseline. Check out the description. Should be same Right?